test/test_transform.cpp

```diff
@@ -2112,6 +2114,7 @@ TEST(Transform_Dictionary)
             auto dict0 = obj0.get_dictionary("dict");
             auto dict1 = obj1.get_dictionary("dict");
 
+            dict0.erase("key");
```

Comment:
Is this instruction now being dropped, since `"key"` has already been erased by the other client? Whereas it would've thrown an exception before?

Reply:
it's not dropped, it actually wins the conflict resolution. But when applying it it does not cause the crash anymore. The issue is when a remote erase comes later than a local erase (which is the case here)